tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestAnnotations.java

```diff
@@ -679,15 +694,88 @@ public void testSymbols() {
      */
     public void testCompleteness() {
         UnicodeSet allRgiNoEs = Emoji.getAllRgiNoES();
-        UnicodeSet namesFound = new UnicodeSet();
-        UnicodeSet searchKeywordsFound = new UnicodeSet();
 
         // get both regular and derived emoji
+        for (String file :
+                List.of("en.xml")) { // for testing, can add others like "de.xml", "fr.xml"
+            UnicodeSet namesFound = new UnicodeSet();
+            UnicodeSet searchKeywordsFound = new UnicodeSet();
+            fillNamesAndSearchKeywords(
+                    file, namesFound, searchKeywordsFound); // freezes the results
+
+            warnln(
+                    Joiner.on('\t')
+                            .join(
+                                    "FYI, RGI:",
```

Comment:
"RGI: Recommended for General Interchange" -- I had to do a web search to find that